tests/src/common/Wsk.scala

```diff
@@ -858,11 +858,13 @@ object WskAdmin {
     /**
      * returns user given the auth key
      */
-    def getUser(authKey: String): String = {
+    def getUser(authKey: String): (String, String) = {
         val wskadmin = new RunWskAdminCmd {}
         val user = wskadmin.cli(Seq("user", "whois", authKey)).stdout.trim
         assert(!user.contains("Subject id is not recognized"), s"failed to retrieve user from authkey '$authKey'")
-        user
+
```

Comment:
a thought you might have considered `--subject` and `--namespace` parameters to select/project only that value (would be useful for bootstrapping wskadmin in other shell commands although the one i use most is to get a key)... it's ok to consider and reject.